### PR TITLE
Sprint 14 (dev->stg)

### PR DESCRIPTION
**Pull request by afrasier**
Includes:
- Many SOAP fixes for DoS
- Glossary CRUD
- OBC ID data
- Performance testing code (lets us boost our position count to a high number, and generate a high number of clients)
  - With 6000 positions we still execute a FTS over 10 columns in 2 seconds. We can get faster, but that's pretty good.

### CODE REVIEW

File: talentmap_api/common/management/commands/load_obc_ids.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+import logging
+import csv
+
+from talentmap_api.common.xml_helpers import CSVloader
+from talentmap_api.organization.models import Post, Country
+
+
+class Command(BaseCommand):
+    help = 'Loads a CSV into a supported model'
```
Comment (jseppi) [inline talentmap_api/common/management/commands/load_obc_ids.py]: `help` text from other command. Mistaken copy-pasta probably.

File: talentmap_api/common/management/commands/load_obc_ids.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        super(Command, self).__init__(*args, **kwargs)
+
+        self.modes = {
+            'post': Post,
+            'country': Country,
+        }
+
+    def add_arguments(self, parser):
+        parser.add_argument('file', nargs=1, type=str, help="The CSV file to load")
+        parser.add_argument('type', nargs=1, type=str, choices=self.modes.keys(), help="The type of data in the CSV")
```
Comment (jseppi) [inline talentmap_api/common/management/commands/load_obc_ids.py]: more copy-pasta

File: talentmap_api/common/management/commands/load_obc_ids.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+    def handle(self, *args, **options):
+        # Parse the CSV
+        with open(options['file'][0], 'r') as csv_file:
+            model = self.modes[options['type'][0]]
```
Comment (jseppi) [inline talentmap_api/common/management/commands/load_obc_ids.py]: is this command actually run for multiple models, or just `Post`?

File: talentmap_api/common/management/commands/create_demo_environment.py [hunk 4]
```diff
@@ -21,6 +25,212 @@ class Command(BaseCommand):
     def __init__(self, *args, **kwargs):
         super(Command, self).__init__(*args, **kwargs)
 
+        self.first_names = [
```
[the remark this answers is not in the document]
Comment (jseppi) [reply talentmap_api/common/management/commands/create_demo_environment.py]: (just a suggestion, not necessary if you don't want to bring in the extra dep)

[the remark this answers is not in the document]
Comment (afrasier) [reply talentmap_api/common/management/commands/create_demo_environment.py]: Hey that's neat. I'll save that for next time.

File: talentmap_api/common/management/commands/load_obc_ids.py (hunk 3 above)
Comment (afrasier) [reply talentmap_api/common/management/commands/load_obc_ids.py]: Post and Country.

**Pull request merged**